src/Snippets/README.md

```diff
@@ -0,0 +1,12 @@
+# Snippet
+It provides convenience to users who create `.cfg` files using vscode (text editor).
+
+It is automatically completed by creating a prefix for each tool in `.cfg` file.

```

Comment:
Keywords are auto completed by the prefix of each tool of `.cfg` file.`
?

Reply:
May I change the 4th line to this?

Reply:
Okay, I will revise it!